03 Writing Algorithms/14 Datasets/02 QuantConnect/01 Binance Crypto Future Margin Rate Data/03 Getting Started.html

```diff
@@ -1,20 +1,24 @@
 <p>The following snippet demonstrates how to request data from the Binance Crypto Future Margin Rate dataset:</p>
 
 <div class="section-example-container">
-<pre class="python">def initialize(self) -&gt; None:
+<pre class="python"># Get data from the Binance crypto future margin rate dataset.
+def initialize(self) -&gt; None:
+    # Use set_brokerage_model method with your own brokerage name.
     self.set_brokerage_model(BrokerageName.BINANCE_FUTURES, AccountType.MARGIN)
     self.set_brokerage_model(BrokerageName.BINANCE_COIN_FUTURES, AccountType.MARGIN)
 
     self.crypto_future_symbol = self.add_crypto_future("BTCBUSD", Resolution.MINUTE).symbol
 </pre>
-<pre class="csharp">private Symbol _cryptoFutureSymbol;
+<pre class="csharp">// Get data from the Binance crypto future margin rate dataset.
+private Symbol _cryptoFutureSymbol;
 
 public override void Initialize
 {
+    // Use set_brokerage_model method with your own brokerage.
     SetBrokerageModel(BrokerageName.BinanceFutures, AccountType.Margin);
     SetBrokerageModel(BrokerageName.BinanceCoinFutures, AccountType.Margin);
 
-    // perpetual futures does not have a filter function
+    // Perpetual futures does not have a filter function.
```

Comment:
@sylver911 Please don't document the /Datasets folder its automatically generated